.github/workflows/ci_pr.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
   build-xenial:
 
-    runs-on: ubuntu-18.04
```

Comment:
Ha! I was just about to comment these jobs out with this comment.

(Not disagreeing with upgrading the `runs-on` version, which is orthogonal and makes perfect sense.)

```yaml
  ## Temporarily disabling build-xenial and build-bionic tests because
  ## these platforms are deprecated. Focal is already our main
  ## development platform, and we expect future ISAAC ISS activities to
  ## occur after the Astrobees on ISS are updated to focal (target
  ## upgrade date 2023/08).
```

Reply:
Yeah I should have put this in a separate PR and merged it already, but I should be done with this PR today :)